react/features/base/react/components/NavigateSectionList.js

```diff
@@ -93,7 +93,7 @@ class NavigateSectionList extends Component<Props> {
 
         return (
             <SectionList
-                ListEmptyComponent = { renderListEmptyComponent }
+                ListEmptyComponent = { renderListEmptyComponent() }
```

Comment:
Was this change necessary? SectionList doc says ListEmptyComponent can be a render function.